modules/porous_flow/doc/content/bib/porous_flow.bib

```diff
@@ -313,3 +326,30 @@ @article{warrick1990
   pages = {253--258},
   year = {1990}
 }
+
+@article{forsyth1995,
+  author = {P. A. Forsyth, Y. S Wu and K. Pruess},
```

Comment:
This reference isn't rendered correctly. Perhaps it needs an 'and' between Forsyth and Wu instead of a comma?